### PR TITLE
Mock Test for AWS Batch Service. AWS StreamtoS3Buffer (incomplete)

### PR DESCRIPTION
**Pull request by chiajieh**

### CODE REVIEW

File: backend/src/main/java/com/ad1/loggenerator/service/implementation/AWSStreamService.java [hunk 1]
```diff
@@ -183,6 +183,8 @@ public void streamToS3(SelectionModel selectionModel, StreamTracker streamJobTra
         // Count the log lines saved to the bucket object, set url and the log count to the stream job tracker
         streamJobTracker.setLogCount(awsLogService.getLogCount(s3Client, s3Object, bucketName, key));
         streamJobTracker.setStreamObjectURL(objectURL);
+        if (streamJobTracker.getStreamObjectURL() == null)
+            streamJobTracker.setStatus(JobStatus.FAILED);
```
Comment (andymbwu) [inline backend/src/main/java/com/ad1/loggenerator/service/implementation/AWSStreamService.java]: This check should be moved higher up, so we can skip initializing the s3Client if it fails

Comment (chiajieh) [reply backend/src/main/java/com/ad1/loggenerator/service/implementation/AWSStreamService.java]: Okay, thanks. I will do that.

File: backend/src/test/java/com/ad1/loggenerator/service/AWSBatchServiceTest.java
Comment (andymbwu) [inline backend/src/test/java/com/ad1/loggenerator/service/AWSBatchServiceTest.java:1]: Tests look good here. Random blank lines should be removed.

Comment (chiajieh) [reply backend/src/test/java/com/ad1/loggenerator/service/AWSBatchServiceTest.java:1]: Okay, thank you. I will do that.

**Comment by chiajieh** [comment]
New pull request created with changes effected

**Pull request closed without merging**